e2e/tests/cohorts/cohort-create.spec.ts

```diff
@@ -164,26 +164,6 @@ describe('Create Cohorts Test', () => {
     expect(await dataPage.findCohortCard(cohortName)).toBeFalsy();
   });
 
-  test('Create cohort from whole genome variant', async () => {
-    await loadWorkspaceUrl(page, workspaceName);
-
-    const dataPage = new WorkspaceDataPage(page);
-    const cohortBuildPage = await dataPage.clickAddCohortsButton();
-
-    const group1 = cohortBuildPage.findIncludeParticipantsGroup('Group 1');
-    await group1.includeWholeGenomeVariant();
-    const group1Count = await group1.getGroupCount();
-    const totalCount = await cohortBuildPage.getTotalCount();
-    expect(group1Count).toEqual(totalCount);
-    expect(Number.isNaN(group1Count)).toBe(false);
-
-    // Save new cohort.
-    await cohortBuildPage.createCohort();
-
-    // Delete cohort in Cohort Build page.
-    await new CohortActionsPage(page).deleteCohort();
-  });
```

Comment:
Removed this test because creating cohort with genomic variant is going to be in the new test `genomic-extraction-to-vcf.spec`.